shell/browser/api/electron_api_web_contents.h

```diff
@@ -91,6 +92,11 @@ class OffScreenWebContentsView;
 
 namespace api {
 
+using DevicePermissionMap = std::map<
+    int,
+    std::map<content::PermissionType,
+             std::map<url::Origin, std::vector<std::unique_ptr<base::Value>>>>>;
```

Comment:
There is no need to store `std::unique_ptr` in a vector.

```suggestion
             std::map<url::Origin, std::vector<base::Value>>>>;
```